doc_internal/cmds_tags.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import argparse, glob, itertools, re, shutil, os, sys
+import subprocess
+import shlex
+
+
+cmds_reg = re.compile(r'.refitem *cmd[a-z]* *')
+tags_obsolete_reg = re.compile(r"^.*option *type=.obsolete.*id='([^']*)'.*")
+tags_active_reg = re.compile(r"^.*option .*id='([^']*)'.*")
+
+
+def xopen(fname, mode='r', encoding='utf-8'):
+    '''Unified file opening for Python 2 an Python 3.
+
+    Python 2 does not have the encoding argument. Python 3 has one.
+    '''
+
+    if sys.version_info[0] == 2:
+        return open(fname, mode=mode) # Python 2 without encoding
+    else:
+        return open(fname, mode=mode, encoding=encoding) # Python 3 with encoding
+
+
+def xpopen(cmd, cmd1="",encoding='utf-8-sig', getStderr=False):
+    '''Unified file pipe opening for Python 2 an Python 3.
+
+    Python 2 does not have the encoding argument. Python 3 has one. and
+    '''
+
+    if sys.version_info[0] == 2:
+        return os.popen(cmd).read() # Python 2 without encoding
+    else:
+        if (getStderr):
+            proc = subprocess.Popen(shlex.split(cmd1),stdout=subprocess.PIPE,stderr=subprocess.PIPE,encoding=encoding) # Python 3 with encoding
+            return proc.stderr.read()
+        else:
+            proc = subprocess.Popen(shlex.split(cmd),stdout=subprocess.PIPE,stderr=subprocess.PIPE,encoding=encoding) # Python 3 with encoding
+            return proc.stdout.read()
+
+
+def get_versions():
+    raw_versions = xpopen('git tag')
+    versions = []
+    strip_versions = []
+    versions.append("Release_1_8_20")
+    pre = "Release_1_9_"
```

Comment:
will this continue to work for future version tags like e.g. `Release_1_10_0` or `Release_2_0_0`?

Reply:
No at that moment we have to one of the following
1. add another loop over `pre` with the new prefixes 
2. process the the first part after Release so we get the major and minor version number
3. move the "1.9" releases into the md file and change `Release_1_9_` in  `Release_1_10_` / `Release_2_0_` (and adjust `versions.append("Release_1_8_20")` as well

For this version I didn't want to make it too complicated, probably the second option would be the best

Reply:
Added functionality for versions like 1.10.0 and 2.0.0